hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -131,6 +131,9 @@
       "hbase.assignment.maximum.attempts";
   private static final int DEFAULT_ASSIGN_MAX_ATTEMPTS = Integer.MAX_VALUE;
 
+  public static final String FAILED_OPEN_ATTEMPTS = "hbase.assignment.failed.open.attempts";
```

Comment:
Srrory for saying again, but maybe name it

ASSIGN_RETRY_IMMEDIATELY_MAX_ATTEMPTS?